src/radical/entk/execman/rp/resource_manager.py

```diff
@@ -171,24 +173,14 @@ def _pilot_state_cb(pilot, state):
 
             self._prof.prof('rreq submitted', uid=self._uid)
 
-            shared_staging_directives = list()
```

Comment:
Why this and the following is now not required?

Reply:
` 'input_staging' : self._shared_data` will replace the `stage_in` step.